lib/routes/account.js

```diff
@@ -441,6 +446,7 @@ module.exports = function (
       handler: function (request, reply) {
         log.begin('Account.RecoveryEmailResend', request)
         var sessionToken = request.auth.credentials
+        db.updateSessionToken(sessionToken, request.headers['user-agent'])
```

Comment:
It may be worth a comment to say that we're deliberately not waiting for this to return before continuing, and/or a wrapper with a suggestive name like `updateSessionTokenInBackground` to make it more obvious.